src/features/landing/components/ForWhoSection/ForWhoSection.tsx

```
@@ -30,23 +70,33 @@ function ForWhoSection(): React.ReactElement {
-                src={desktopProps.src as string}
-                width={desktopProps.width}
-                height={desktopProps.height}
-                alt={desktopProps.alt}
-                loading="lazy"
-              />
-            </picture>
+          <Box sx={styles.container}>
+            <Box sx={styles.svg}>
+              {svgGroup.map(({ src, sx }, index) => (
+                <Box
+                  key={`decorative-group1-${index}`}
```

ESLint

[error] Do not use Array index in keys

(react/no-array-index-key)